spec/integration/riak/crdt_spec.rb

```diff
@@ -219,6 +219,16 @@
         expect(subject.maps['road'].counters['speedbumps'].value).to eq 4
         expect(subject.maps['road'].sets['signs'].include? 'yield').to be
       end
+
+      it 'deletes nested inner-map' do
+        bag_map = subject.maps['bag']
+        inner_map = bag_map.maps['123']
+        inner_map.registers['name'] = 'f1'
+
+        expect(subject.maps['bag'].maps['123'].registers['name']).to eq 'f1'
+
+        expect{ bag_map.maps.delete('123') }.to_not raise_error
```

Comment:
Can you add an expectation for `bag_map.maps['123'].registers['name']` `to_not be` or something to that effect?